src/main/common/utils/numericUtils.ts

```diff
@@ -14,3 +14,17 @@ export function toNumberOrUndefined (value: any): number {
 
   return isNaN(numberVal) ? undefined : numberVal
 }
+
+export function containsAThousandSeparator (input: string): boolean {
```

Comment:
Please do not expose that function, no point in having it and then using it in tens of places. Change `toNumberOrUndefined` which is a central point of deserialising numbers.

A `toNumberOrUndefined` should use numeral as an improvement on top of dumb `_.toNumber(strVal)`

Plase do the following:
1. move `containsAThousandSeparator` to `common/utils/numericUtils.ts`, make it private and reuse within `toNumberOrUndefined` as an improvement to that method
2. Use `toNumberOrUndefined ` in all places that have:

```const amount = value.amount ? !value.amount.toString().includes(',') || containsAThousandSeparator(value.amount.toString())  ? numeral(value.amount).value() : value.amount : undefined```

3. Add tests that ensure deserialisation logic works fine with thousand separators, nobody is interested in knowing how `containsAThousandSeparator` works.